docs/zh/latest/admin-api.md

```diff
@@ -116,7 +116,7 @@ APISIX 在 v3 版本对响应体做了以下调整：
 - 移除旧版本响应体中的 `action` 字段；
 - 调整获取资源列表时的响应体结构，新的响应体结构示例如下：
 
-返回单个资源：
+  * 返回单个资源：
```

Comment:
It's not necessary, just remove then blank before ` ``` `

Reply:
Please review again, thanks. @monkeyDluffy6017  @SylviaBABY